src/server/scripts/Spells/spell_priest.cpp

```diff
@@ -1255,6 +1257,36 @@ class spell_pri_guardian_spirit : public AuraScript
     }
 };
 
+// 421558 - Heaven's Wrath
+class spell_pri_heavens_wrath : public AuraScript
+{
+    bool Validate(SpellInfo const* /*spellInfo*/) override
+    {
+        return ValidateSpellInfo
+        ({
+            SPELL_PRIEST_HEAVENS_WRATH,
```

Comment:
Don't put the spell you are attaching script to inside Validate (it's always validated internally before even reaching this function)

Reply:
so i should put eveything else i use besides the spell im attaching to